src/types/python/excelreader.rs

```diff
@@ -250,12 +318,18 @@ impl ExcelReader {
                     if self.sheet_names.contains(&name) {
                         Ok(name)
                     } else {
-                        Err(FastExcelErrorKind::SheetNotFound(IdxOrName::Name(name.clone())).into()).with_context(||  {
-                            let available_sheets = self.sheet_names.iter().map(|s| format!("\"{s}\"")).collect::<Vec<_>>().join(", ");
-                            format!(
-                                "Sheet \"{name}\" not found in file. Available sheets: {available_sheets}."
-                            )
-                        })
+                        Err(FastExcelErrorKind::SheetNotFound(IdxOrName::Name(name.clone())).into())
+                            .with_context(|| {
+                                let available_sheets = self
+                                    .sheet_names
+                                    .iter()
+                                    .map(|s| format!("\"{s}\""))
+                                    .collect::<Vec<_>>()
+                                    .join(", ");
+                                format!(
+								"Sheet \"{name}\" not found in file. Available sheets: {available_sheets}."
+							)
+                            })
```

Comment:
Curious, it seems this might be caused by the diff visualization. I've check my IDE (RustRover) and the online GH IDE (triggered by pressing `.`) and it seemed fine. Let me know if this is the case for you too.

Reply:
Looks fine now, thanks. Seems like https://github.com/ToucanToco/fastexcel/pull/282/commits/1964c2edc032990cc1390d10628745d83b541fee solved it